src/redux/cofigureStore.js

```diff
@@ -0,0 +1,36 @@
+import { createStore } from 'redux';
+
+const initialState = {
+  books: [
+    {
+      id: 0,
+      categorie: 'Action',
+      title: 'The Hunger Games',
+      author: 'Suzanne Collins',
+    },
+    {
+      id: 1,
+      categorie: 'Science Fiction',
+      title: 'Dune',
+      author: 'Frank Herbert',
+    },
+    {
+      id: 2,
+      categorie: 'Economy',
+      title: 'Capital in the Twenty-First Century',
+      author: 'Suzanne Collins',
+    },
+  ],
+};
+
+function rootReducer(state = initialState, action) {
+  switch (action.type) {
+    default:
+      return state;
+  }
+}
+
+const store = createStore(rootReducer);
+window.store = store;
+
+export default store;
```

Comment:
- [ ] And also you don't need this for the momoent.

Reply:
I need this to use in the initial State, but I will change it on the second Milestone.

Reply:
- [ ] You don't need a reducer yet, because your list of books should be inside your `BookList` component, kindly change this 😄